packit_service/constants.py

```diff
@@ -72,11 +72,14 @@
 COPR_API_SUCC_STATE = 1
 COPR_API_FAIL_STATE = 2
 
+# Retry 2 times
 DEFAULT_RETRY_LIMIT = 2
 # Retry more times for outages
 DEFAULT_RETRY_LIMIT_OUTAGE = 5
-# retry in 3s, 6s
-DEFAULT_RETRY_BACKOFF = 3
+# retry in 0-7s, 0-14s, 0-28s, 0-48s, 0-96s
+# because jitter is enabled by default, celery makes these retries random:
+# https://docs.celeryq.dev/en/latest/userguide/tasks.html#Task.retry_jitter
+DEFAULT_RETRY_BACKOFF = 7
```

Comment:
Because of the jitter, it can still end really soon when we have bad luck, can't it?

(Just wondering if jitter helps here.)

Reply:
yes, that's true, though very unlikely, we'd need to roll 1-5s 6-7 times in a row; and especially once we get to 0-28, 0-48, 0-96 that's becoming 0.001% :D

completely agree with you though if we could specify the jitter function that would be awesome

Reply:
tl;dr celery's reasoning for jitter is not to cause a barrage of requests in a given timeframe and rather spread these, which makes sense

I'm totally up to jitter=false in the next PR to try it out

Reply:
You are right, it's really unlikely and if they hit at least one higher value, we are good.

Reply:
And we can improve if needed.